samples/dotnet/RedisOutputBinding/StreamTriggerDeleteEntry.cs

```diff
@@ -10,12 +10,12 @@ internal class StreamTriggerDeleteEntry
         [FunctionName(nameof(StreamTriggerDeleteEntry))]
         public static void Run(
             [RedisStreamTrigger(Common.connectionStringSetting, "streamTest2")] StreamEntry entry,
-            [Redis(Common.connectionStringSetting, "XDEL")] out string[] result,
+            [Redis(Common.connectionStringSetting, "XDEL")] out string result,
             ILogger logger)
         {
             logger.LogInformation($"Stream entry from key 'streamTest2' with Id '{entry.Id}' and values '" +
                 JsonConvert.SerializeObject(entry.Values.ToDictionary(x => x.Name.ToString(), x => x.Value.ToString())));
-            result = new string[] { "streamTest2", entry.Id.ToString() };
+            result = "streamTest2 " + entry.Id.ToString();
```

Comment:
Will the result ever be parsed (split by space) again downstream? If yes, will spaces, which come from server like the space in the key name, cause surprise?

Reply:
The extension is the only one splitting by space. In this case we are only sending a command to Redis and not caring about the value that the command returns, so there won't be anything special from the server.
But if there is a space in the key name, our logic will split the key and the command won't work correctly. Customers will have to use keys that don't contain any spaces in them. This is the same limitation that the input binding has because it also only takes a single string for the entire command that is space-delimited

Reply:
Okay, as long as users are aware of this.